internal/storage/storage_handle.go

```diff
@@ -78,7 +69,52 @@ func createGRPCClientHandle(ctx context.Context, clientConfig *storageutil.Stora
 	clientOpts = append(clientOpts, option.WithGRPCConnectionPool(clientConfig.GrpcConnPoolSize))
```

Comment:
clientConfig.GrpcConnPoolSize needs to be shared between control client and data client i.e if user has set clientConfig.GrpcConnPoolSize to be 100, then control client can have 2 and data clients can have 98, so total number of grpc connections are equal to this value. cc @raj-prince @sethiay

Reply:
@Tulsishah we can keep this change as-is for now, will refactor once we reach consensus with go sdk team on this setting